### PR TITLE
chore: remove deprecated methods

### PR DESCRIPTION
**Pull request by taka-oyama**
- `Connection::runDdl()` and `Connection::runDdls()` has been removed. Use `Connection::runDdlBatch()` instead.
- Trait `ManagesStaleReads` has been removed (which contained `cursorWithTimestampBound()` and `selectWithTimestampBound()`. Use `selectWithOptions()` instead).
- `Blueprint::interleave()` and `IndexDefinition::interleave()` now throw an error instead of a deprecation.

### CODE REVIEW

File: src/Concerns/ManagesDataDefinitions.php [hunk 1]
```diff
@@ -30,28 +30,6 @@ trait ManagesDataDefinitions
...
     abstract public function getSpannerDatabase(): Database;
 
-    /**
-     * @deprecated use runDdlBatch() instead
-     * @param string $ddl
-     * @return LongRunningOperation
-     */
-    public function runDdl(string $ddl): LongRunningOperation
-    {
-        trigger_deprecation('colopl/laravel-spanner', '5.2', 'runDdl() is deprecated. Use runDdlBatch() instead.');
```
Comment (tyabu12) [inline src/Concerns/ManagesDataDefinitions.php:40]: `use function trigger_deprecation;` is no longer used.

**Review by tyabu12 — CHANGES_REQUESTED** [review]

File: src/Concerns/ManagesDataDefinitions.php (hunk 1 above)
Comment (taka-oyama) [reply src/Concerns/ManagesDataDefinitions.php:40]: deleted.

**Review by tyabu12 — APPROVED** [review]
LGTM

**Comment by taka-oyama** [comment]
Thanks for reviewing

**Pull request merged**